packages/ui/src/components/common/CustomAddressModal.tsx

```diff
@@ -118,7 +121,7 @@ export const CustomAddressModal: FC<Props> = ({
                 height: 48
               }}
               placeholder={
-                isSolanaSwap ? 'Enter Solana address' : 'Address or ENS'
+                isSvmSwap ? `Enter ${toChain} address` : 'Address or ENS'
```

Comment:
Does this need to be toChain.displayName?

Reply:
thanks!